app/experimenter/templates/experiments/rapid.html

```diff
@@ -2,7 +2,15 @@
 {% load static%}
 
 
+{% load static %}
+
 {% block main %}
+  {% if FEATURE_ANALYSIS %}
+    <script src="{% static "assets/visualization/bundle.js" %}"></script>
+    <link rel="stylesheet" href="{% static "assets/visualization/bundle.css" %}" />
+  {% endif %}
+
+  <link rel="stylesheet" href="{% static "assets/css/index.css" %}">
```

Comment:
Is this css for the visualization?

Reply:
yes

Reply:
That path/name might collide, is there any way to to something like `assets/css/visualization/index.css`?

Reply:
Or `assets/visualization/index.css`?

Reply:
sure

Reply:
Actually realized this wasn't needed at all